index.html

```diff
@@ -89,6 +89,212 @@ <h1 class="presentation">
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+              <ul class="work__lang">
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+              </ul>
+            </div>
+            <button type="button" class="btn btn-enabled">See project</button>
+          </div>
+        </article>
+        <article class="works">
+          <div class="works__image">
+            <img src="images/mobile/3.png" alt="work 3" class="img3" />
+          </div>
+          <div class="work__description">
+            <h2>Tonic</h2>
+            <div class="short_desc">
+              <h4>CANOPY</h4>
+              <span></span>
+              <h4>Back end Dev</h4>
+              <span></span>
+              <p>2015</p>
+            </div>
+            <div class="long_desc">
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+              <ul class="work__lang">
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+              </ul>
+            </div>
+            <button type="button" class="btn btn-enabled">See project</button>
+          </div>
+        </article>
+        <article class="works">
+          <div class="works__image">
+            <img src="images/mobile/4.png" alt="work 4" class="img4" />
+          </div>
+          <div class="work__description">
+            <h2>Multi-Post Stories</h2>
+            <div class="short_desc">
+              <h4>CANOPY</h4>
+              <span></span>
+              <h4>Back end Dev</h4>
+              <span></span>
+              <p>2015</p>
+            </div>
+            <div class="long_desc">
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+              <ul class="work__lang">
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+              </ul>
+            </div>
+            <button type="button" class="btn btn-enabled">See project</button>
+          </div>
+        </article>
+      </section>
+      <section class="about-me" id="about">
+        <div class="about-me-container">
+          <div class="about-me__title">
+            <h2>About Myself</h2>
+          </div>
+          <div class="about-me__description">
+            <p>
+              Hello I’m a software developer! I can help you build a product ,
+              feature or website Look through some of my work and experience! If
+              you like what you see and have a project you need coded, don’t
+              hestiate to contact me.
+            </p>
+          </div>
+          <a href="#" class="medias">let's connect</a>
```

Comment:
- [ ] Good job on this but this is supposed to be a header tag `hx`. Kindly change it to be a header tag